spec/moronepo_run_action_spec.rb

```diff
@@ -0,0 +1,35 @@
+describe Fastlane::Actions::MoronepoRunAction do
+  describe 'moronepo_activate' do
+    it 'should run proper shell command' do
+      expect(Fastlane::Actions)
+        .to receive(:sh)
+        .with("flutter pub global run moronepo run -- flutter packages get")
+
+      ActionRunner.moronepo_run("arguments: 'flutter packages get'")
+    end
+
+    it 'should run shell command with working_directory param' do
+      expect(Fastlane::Actions)
+        .to receive(:sh)
+        .with("flutter pub global run moronepo --working-directory nepo_dir run -- flutter packages get")
```

Comment:
`nepo_dir`? :)

Reply:
It was first name that i have in mind. The second was `moron_dir`